src/NuGet.Core/NuGet.Protocol.Core.v3/HttpSource/HttpRequestMessageFactory.cs

```diff
@@ -42,22 +46,26 @@ public static HttpRequestMessage Create(HttpMethod method, string requestUri, IL
         /// </summary>
         /// <param name="method">Desired HTTP verb</param>
         /// <param name="requestUri">Request URI</param>
-        /// <param name="log">Logger instance to be attached</param>
+        /// <param name="configuration">The request configuration</param>
         /// <returns>Instance of <see cref="HttpRequestMessage"/></returns>
-        public static HttpRequestMessage Create(HttpMethod method, Uri requestUri, ILogger log)
```

Comment:
Besides adding new method I'd keep original factory method signature introduced solely for convenience of use as a shortcut. This also would result in less code changes.